slideview/src/main/java/ng/max/slideview/SlideView.java

```diff
@@ -157,6 +151,43 @@ public void setSlideBackgroundColor(ColorStateList color) {
         Util.setDrawableColor(slideBackground, color.getDefaultColor());
     }
 
+    public void setStokeColor(@ColorInt int color) {
+        Util.setDrawableStroke(slideBackground, color);
+    }
+
+    public void setReverseSlide(boolean reverseSlide) {
+        _setReverseSlide(reverseSlide);
+    }
+
+    private void _setReverseSlide(boolean reverseSlide) {
+        mReverseSlide = reverseSlide;
+        if (reverseSlide) {
+            slider.setRotation(180);
+            LayoutParams params = ((LayoutParams) slideTextView.getLayoutParams());
+            params.addRule(RelativeLayout.ALIGN_PARENT_RIGHT, 0);
+            params.addRule(RelativeLayout.ALIGN_PARENT_LEFT);
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
+                params.addRule(RelativeLayout.ALIGN_PARENT_END, 0);
+                params.addRule(RelativeLayout.ALIGN_PARENT_START);
+            }
+            slideTextView.setLayoutParams(params);
+        } else {
+            slider.setRotation(0);
+            LayoutParams params = ((LayoutParams) slideTextView.getLayoutParams());
+            params.addRule(RelativeLayout.ALIGN_PARENT_LEFT, 0);
+            params.addRule(RelativeLayout.ALIGN_PARENT_RIGHT);
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
+                params.addRule(RelativeLayout.ALIGN_PARENT_START, 0);
+                params.addRule(RelativeLayout.ALIGN_PARENT_END);
+            }
+            slideTextView.setLayoutParams(params);
+        }
+    }
+
+    public boolean getReverseSlide() {
```

Comment:
After the field is renamed to `reversed`, this getter should be renamed to `isReversed()`.